packages/nc-gui-v2/components/smartsheet-column/EditOrAdd.vue

```diff
@@ -1,5 +1,6 @@
 <script lang="ts" setup>
-import { UITypes } from 'nocodb-sdk'
+import { UITypes, isVirtualCol } from 'nocodb-sdk'
+import RollupOptions from './RollupOptions.vue'
```

Comment:
I think you don't need to import it. You can use `SmartsheetColumnRollupOptions` directly as it is imported in `components.d.ts`